apps/devhub/templates/devhub/addons/listing/delete_form.html

```diff
@@ -21,10 +21,10 @@
   {% endif %}
   <p>
     <label>
-      {{ _('Enter your password to confirm your decision:') }}
-      {# see bug 630126 for why this is called 'password-timestamp' #}
-      <input type="password" name="password-{{ timestamp }}" data-name="password" autocomplete="off" required>
```

Comment:
I initially went to remove it but noticed other files using the JS. The delete account view has a similar password prompt so I'm betting it's that. I'll check again once I remove the other password prompt.

I'll pull `timestamp` out of the context though, good call.